app/views/RoomsListView/index.js

```diff
@@ -141,7 +138,9 @@ class RoomsListView extends React.Component {
 		insets: PropTypes.object,
 		queueSize: PropTypes.number,
 		inquiryEnabled: PropTypes.bool,
-		encryptionBanner: PropTypes.string
+		encryptionBanner: PropTypes.string,
+		showAvatar: PropTypes.bool,
+		displayType: PropTypes.string
```

Comment:
Depending on the `displayType` you should modify as well the alignment for the content that appears on swipe, so we prevent this:
![image](https://user-images.githubusercontent.com/36211892/128921171-f5016f5e-3370-4203-8ed7-b94f11635bd1.png)